nixos/modules/misc/ids.nix

```diff
@@ -475,7 +475,7 @@ in
       #minecraft = 114; # unused
       vault = 115;
       #ripped = 116; # unused
-      murmur = 117;
+      mumble-server = 117;
```

Comment:
Why do we need a persistent uid/gid at all? I believ we could be fine with DynamicUser. @hax404?

Reply:
See also NixOS/rfcs#52. Additionally I think just changing the name here would break things, because `/var/lib/nixos/{u,g}id-map` still maps `murmur` to 117. Not sure about the consequences.